### PR TITLE
[10.0][NEW] account_bank_statement_import_move_line

### PR DESCRIPTION
**Pull request by luismontalba**
Account Bank Statement Import Move Line
==================================
This module adds a button to bank statement form view to open a wizard to allow filtering, selecting and importing lines form journal items into the bank statement.

Usage
--------
* Go to Invoicing > Dashboard.
* Create a new bank statement from a bank journal.
* Click button "Import Journal Items".
* Select filtering options.
* Click on button "Add All Move Lines" to auto-select the move lines matching the selected criteria or click on "Add an item" to manually select the move lines filtered by the criteria.
* Click on button "Create Statement Lines".

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Next time, let 9.0 commit, and create another commit with the 10.0 migration.

@cubells @chienandalu please review

File: account_bank_statement_import_move_line/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/174/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/bank-statement-import/issues>`_. In case of trouble,
+please
+check there if your issue has already been reported. If you spotted it first,
```
Comment (cubells) [inline account_bank_statement_import_move_line/README.rst]: Reformat this line break

File: account_bank_statement_import_move_line/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,180 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* account_bank_statement_import_move_line
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 9.0c\n"
```
Comment (cubells) [inline account_bank_statement_import_move_line/i18n/es.po]: Translation is not updated?

File: account_bank_statement_import_move_line/tests/test_account_bank_statement_import_move_line.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        cls.a_receivable = cls.env['account.account'].create({
+            'code': 'TAA',
+            'name': 'Test Receivable Account',
+            'internal_type': 'receivable',
+            'user_type_id': cls.account_type.id,
+        })
+        cls.partner = cls.env['res.partner'].create({
+            'name': 'Test Partner 2',
+            'parent_id': False,
+            'account_id': cls.a_receivable.id,
```
Comment (cubells) [inline account_bank_statement_import_move_line/tests/test_account_bank_statement_import_move_line.py]: There is not exist a field "account_id" in res_partner model. Is It necessary that field?

**Review by cubells — CHANGES_REQUESTED** [review]
Minor changes.

**Comment by luismontalba** [comment]
Translation updated and changes proposed done (commit squashed).

**Comment by luismontalba** [comment]
Travis log:

/home/travis/maintainer-quality-tools/travis/travis_install_nightly: line 21:  4721 Segmentation fault      (core dumped) npm install -g eslint
The command "travis_install_nightly" failed and exited with 139 during .
Your build has been stopped.

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test.

**Review by chienandalu — APPROVED** [review]
Tested on runbot :+1: 
Travis error seems to be unrelated

**Comment by pedrobaeza** [comment]
I have relaunched Travis pyltint state and is OK now (a temporary glitch?), so I merge.

**Pull request merged**